File: message.php
<?php

require 'vendor/message.php';

use danog\MadelineProto\API;
use danog\MadelineProto\EventHandler;

class MyEventHandler extends EventHandler
{
    public function onAny($update)
    {
        // Handle different types of updates
        if (isset($update['message'])) {
            $this->onMessage($update['message']);
        }
    }

    public function onMessage($message)
    {
        $user_id = $message['from']['id'];
        $text = "Sender ID: $user_id\nMessage: {$message['text']}";

        // Replace 'YOUR_GROUP_ID' with your actual group ID
        $this->messages->sendMessage(['peer' => 'YOUR_GROUP_ID', 'message' => $text]);

        // Reply to the user
        $this->messages->sendMessage(['peer' => $user_id, 'message' => "Your message has been received and forwarded!"]);
    }
}

$settings = [
    'app_info' => [
        'api_id' => 'your_api_id',
        'api_hash' => 'your_api_hash',
    ],
];

$MadelineProto = new API('session.madeline', $settings);
$MadelineProto->start();

$MadelineProto->setEventHandler(MyEventHandler::class);
$MadelineProto->loop(-1);

?>

File: message.py
import logging
from telegram import Update
from telegram.ext import Updater, CommandHandler, MessageHandler, Filters, CallbackContext

# Replace 'YOUR_BOT_TOKEN' with your actual bot token
TOKEN = 'YOUR_BOT_TOKEN'

# Enable logging
logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', level=logging.INFO)

# Initialize the bot
updater = Updater(token=TOKEN, use_context=True)
dispatcher = updater.dispatcher

# Handler for forwarding messages to the group and replying to the user
def forward_and_reply(update: Update, context: CallbackContext):
    user_id = update.message.from_user.id
    text = f"Sender ID: {user_id}\nMessage: {update.message.text}"

    # Replace 'YOUR_GROUP_ID' with your actual group ID
    context.bot.send_message(chat_id='YOUR_GROUP_ID', text=text, parse_mode='Markdown')

    # Reply to the user
    update.message.reply_text("Your message has been received and forwarded!")

# Register the handler
message_handler = MessageHandler(Filters.text & ~Filters.command, forward_and_reply)
dispatcher.add_handler(message_handler)

# Start the bot
updater.start_polling()
updater.idle()
